src/Config/Redirect.php

```diff
@@ -106,18 +106,26 @@ public function getResult($path)
         }, $path);
 
         // Replace variables with actual data
-        foreach ($this->config->getVariables() as $variable => $data) {
-            $result = str_replace("{@$variable}", ltrim($data, '/'), $result);
+        $variables = $this->config->getVariables();
+        if ($variables) {
```

Comment:
Hmm, just thinking out loud here, but wouldn't it be nicer to do something like `return $this->get('variables') ?: [];` here: https://github.com/SahAssar/boltredirector/blob/master/src/Config/Config.php#L79 and the same for getJits below? That'd basically fix the issue too (but with an actual config check rather than having to have the logic here), right?